crates/voicevox_core_c_api/tests/e2e/testcases/compatible_engine.rs

```diff
@@ -0,0 +1,142 @@
+use std::ffi::CStr;
+
+use assert_cmd::assert::AssertResult;
+
+use crate::{float_assert, snapshots::SNAPSHOTS, Symbols, Utf8Output};
+
+pub(crate) unsafe fn exec(
+    Symbols {
+        initialize,
+        load_model,
+        is_model_loaded,
+        finalize,
+        metas,
+        supported_devices,
+        yukarin_s_forward,
+        yukarin_sa_forward,
+        decode_forward,
+    }: Symbols<'_>,
+) -> anyhow::Result<()> {
+    let metas_json = {
+        let metas_json = metas();
+        let metas_json = CStr::from_ptr(metas_json).to_str()?;
+        metas_json.parse::<serde_json::Value>()?;
+        metas_json
+    };
+
+    let supported_devices = {
+        let supported_devices = supported_devices();
+        CStr::from_ptr(supported_devices)
+            .to_str()?
+            .parse::<serde_json::Value>()?
+    };
+
+    assert!(initialize(false, 0, false));
+
+    assert!(!is_model_loaded(SPEAKER_ID));
+    assert!(load_model(SPEAKER_ID));
+    assert!(is_model_loaded(SPEAKER_ID));
+
+    // "テスト"
```

Comment:
めちゃめちゃ細かいですが、テストという文字列でテストすることがわかるとより丁寧かも！
```suggestion
    // テスト用テキストは"t e s u t o"
```